### PR TITLE
Added tech preview and optional secured cluster settings for Operator

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-15875

Preview: 
- https://61032--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/install-secured-cluster-config-options-ocp.html#per-node-settings_install-secured-cluster-config-options-ocp
- https://61032--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/install-secured-cluster-ocp.html#secured-cluster-services-config_install-secured-cluster-ocp

Cherrypick:
- `rhacs-docs-4.1`

### CODE REVIEW

File: installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="install-secured-cluster-config-options-ocp"]
+= Optional - Configuring Secured cluster configuration options for RHACS using the Operator
+:context: install-secured-cluster-config-options-ocp
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[role="_abstract"]
+This topic provides information about optional configuration options that you can configure using the Operator.
```
Comment (skopacz1) [inline installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc:10]: Suggestion:

"optional configuration options that you can configure" reads a little awkward to me, how about the following:

"This topic provides information about optional configurations that you can make using the Operator."

File: modules/secured-cluster-configuration-options-operator.adoc [hunk 2]
```diff
@@ -172,6 +172,7 @@ The default value is `EBPF`.
 Red Hat recommends using `EBPF` for data collection.
 If you select `NoCollection`, Collector does not report any information about the network activity and the process executions.
 Available options are `NoCollection`, `EBPF`, and `CORE_BPF`.
+`CORE_BPF` collection method is a Technology Preview feature only.
```
Comment (skopacz1) [inline modules/secured-cluster-configuration-options-operator.adoc:175]: I think this sentence should start with "The" before `CORE_BPF`.

File: modules/secured-cluster-configuration-options-operator.adoc [hunk 3]
```diff
@@ -192,6 +193,9 @@ If you use the `Slim` image type, you must ensure that your Central instance is
 
 |===
 
+:FeatureName: `CORE_BPF` collection method
```
Comment (skopacz1) [inline modules/secured-cluster-configuration-options-operator.adoc:196]: Same as my other comment re: adding "The" before `CORE_BPF`.

File: modules/secured-cluster-services-config.adoc [hunk 4]
```diff
@@ -70,6 +70,7 @@
 
 | `collector.collectionMethod`
 | Either `EBPF`, `CORE_BPF`, or `NO_COLLECTION`.
+`CORE_BPF` collection method is a Technology Preview feature only.
```
Comment (skopacz1) [inline modules/secured-cluster-services-config.adoc:73]: Same as my other comment re: adding "The" before `CORE_BPF`.

File: modules/secured-cluster-services-config.adoc [hunk 5]
```diff
@@ -270,6 +271,9 @@ Otherwise, you must ensure that Central can access the online probe repository h
 
 |===
 
+:FeatureName: `CORE_BPF` collection method
```
Comment (skopacz1) [inline modules/secured-cluster-services-config.adoc:274]: Same as my other comment re: adding "The" before `CORE_BPF`.

**Review by skopacz1 — COMMENTED** [review]
I left a few suggestions but otherwise looks good!

This is my first week on the peer review squad, please wait until @adellape gives my review a thumbs up or OK before implementing any feedback. Thanks!

File: _topic_map.yml [hunk 6]
```diff
@@ -114,6 +114,8 @@ Topics:
     File: init-bundle-ocp
   - Name: Installing secured cluster services for RHACS on Red Hat OpenShift
     File: install-secured-cluster-ocp
+  - Name: Optional - Configuring Secured cluster configuration options for RHACS using the Operator
```
Comment (adellape) [inline _topic_map.yml:117]: Possibly out of scope, but should these "Optional -" (both in the topic_map and headings) be "Optional: "? While RHACS looks to be the first to actually have it in a topic_map (I see there is also another introduced by an earlier PR), in other openshift-docs branches it looks like we've followed the [IBM Style Guide's guidance on optional steps](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) (which uses colons) for headings, at least.

File: installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc (hunk 1 above)
Comment (adellape) [reply installing/installing_ocp/install-secured-cluster-config-options-ocp.adoc:10]: 👍 or another word like "optional configuration settings", etc.

**Review by adellape — COMMENTED** [review]
@skopacz1 @gaurav-nelson Peer review follow-up complete 👍

File: _topic_map.yml (hunk 6 above)
Comment (gaurav-nelson) [reply _topic_map.yml:117]: Thank you @adellape for the suggestion, I'll get it done as a follow-up PR.

**Comment by gaurav-nelson** [comment]
It is a `4.1 ` only change not needed to cherry-pick into other branches.

**Pull request merged**